src/sortStudents.ts

```diff
@@ -1,16 +1,68 @@
-
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
+  Surname,
+  Age,
+  Married,
+  AverageGrade,
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
+
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  const copyArray: Student[] = [...students];
+
+  switch (sortBy) {
+    case SortType.Name: {
+      copyArray.sort((a, b) => a.name.localeCompare(b.name));
+      break;
+    }
+
+    case SortType.Surname: {
+      copyArray.sort((a, b) => a.surname.localeCompare(b.surname));
+      break;
+    }
+
+    case SortType.Age: {
+      copyArray.sort((a, b) => b.age - a.age);
```

Comment:
```suggestion
      copyArray.sort((firstStudent, secondStudent) => secondStudent.age - firstStudent.age);
```
change namings in all code like this :)